cmd/clusterctl/client/client.go

```diff
@@ -65,6 +65,15 @@ type Client interface {
 	// ProcessYAML provides a direct way to process a yaml and inspect its
 	// variables.
 	ProcessYAML(options ProcessYAMLOptions) (YamlPrinter, error)
+
+	// Interface for alpha features in clusterctl
+	AlphaClient
```

Comment:
@joelanford @alexeldeib Can you help me make sense of why this is flagged as an "incompatible change" with `go-apidiff`? I also tried removing the embedded interface and just putting the func directly in Client interface, but similar issue. 
```
*** Running go-apidiff ***

sigs.k8s.io/cluster-api/cmd/clusterctl/client
  Incompatible changes:
  - AlphaClient.RolloutRestart: added
  Compatible changes:
  - (*clusterctlClient).RolloutRestart: added
  - AlphaClient: added
  - RolloutRestartOptions: added

sigs.k8s.io/cluster-api/cmd/clusterctl/client/rollout
  Compatible changes:
  - ObjectRestarter: added

sigs.k8s.io/cluster-api/cmd/clusterctl/cmd/rollout
  Compatible changes:
  - NewCmdRolloutRestart: added
+ EXIT_VALUE=1
+ set +o xtrace
```
https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_cluster-api/3838/pull-cluster-api-apidiff-main/1319649725397340160/build-log.txt

Reply:
The test is more advisory than required -- it's okay that it fails for PRs to be merged some times.  The PR does make all of the changes mentioned, but they all seem reasonable to me (actually a bit surprised the first one is incompatible and the second one is compatible, would have expected those to be flipped since adding to an existing interface would require implementers to update their code).

Reply:
Ah okay. Unless others see any problems with what I'm doing, I'll just ignore this for now.